README.md

```diff
@@ -178,3 +178,12 @@ LD_LIBRARY_PATH=/path/to/libs ./main
 CGO_ENABLED=1 CGO_LDFLAGS="-L/path/to/libs" go build -tags=duckdb_use_lib main.go
 DYLD_LIBRARY_PATH=/path/to/libs ./main
 ```
+
+## Notes
+
+`TIMESTAMP vs. TIMESTAMP_TZ`
+
+In the C API, DuckDB stores both `TIMESTAMP` and `TIMESTAMP_TZ` as `duckdb_timestamp`, which holds the number of 
+microseconds elapsed since January 1, 1970 UTC. When passing a `time.Time` with a different location to go-duckdb, 
+it'll transform it with `UnixMicro()`, even when using `TIMESTAMP_TZ`. Scanning that value later returns the time in `UTC`, 
```

Comment:
In the C API, DuckDB stores both `TIMESTAMP` and `TIMESTAMP_TZ` as `duckdb_timestamp`, which holds the number of 
microseconds elapsed since January 1, 1970 UTC (i.e., an instant without offset information). When passing a `time.Time` to go-duckdb, 
it will transform it to an instant with `UnixMicro()`, even when using `TIMESTAMP_TZ`. Later, scanning either type of value returns an instant, as SQL types do not model time zone information for individual values.